test/e2e/acl_logging.go

```diff
@@ -0,0 +1,204 @@
+		Eventually(func() (bool, error) {
+			return assertLogs(
+				clientPodScheduledPodName,
+				nsName,
+				denyAllPolicyName,
+				initialDenyACLSeverity)
+		}, maxPokeRetries*pokeInterval, pokeInterval).Should(BeTrue())
+	})
+
+	When("the namespace's ACL logging annotation is updated", func() {
+		const updatedAllowACLLogSeverity = "debug"
+
+		BeforeEach(func() {
+			By(fmt.Sprintf("updating the namespace's ACL logging level to %s", updatedAllowACLLogSeverity))
+
+			namespace, err := fr.ClientSet.CoreV1().Namespaces().Get(context.Background(), nsName, metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred(), "failed to retrieve the namespace")
+			Expect(setNamespaceACLLogSeverity(namespace, updatedAllowACLLogSeverity, updatedAllowACLLogSeverity)).To(Succeed())
+			namespace, err = fr.ClientSet.CoreV1().Namespaces().Get(context.Background(), nsName, metav1.GetOptions{})
+		})
+
+		BeforeEach(func() {
+			By("poking some more...")
+			clientPod := pods[pokerPodIndex]
+			pokedPod := pods[pokedPodIndex]
+
+			framework.Logf(
+				"Poke pod %s (on node %s) from pod %s (on node %s)",
+				pokedPod.GetName(),
+				pokedPod.Spec.NodeName,
+				clientPod.GetName(),
+				clientPod.Spec.NodeName)
+			Expect(
+				pokePod(fr, clientPod.GetName(), pokedPod.Status.PodIP)).To(HaveOccurred(),
+				"traffic should be blocked since we only use a deny all traffic policy")
+		})
+
+		It("the ACL logs are updated accordingly", func() {
+			clientPodScheduledPodName := pods[pokerPodIndex].Spec.NodeName
+			Eventually(func() (bool, error) {
+				return assertLogs(
+					clientPodScheduledPodName,
+					nsName,
+					denyAllPolicyName,
+					updatedAllowACLLogSeverity)
+			}, maxPokeRetries*pokeInterval, pokeInterval).Should(BeTrue())
+		})
+	})
+})
+
+func assertLogs(targetNodeName string, namespace string, policyName string, aclSeverity string) (bool, error) {
+	framework.Logf("collecting the ovn-controller logs for node: %s", targetNodeName)
+	targetNodeLog, err := runCommand([]string{"docker", "exec", targetNodeName, "grep", "acl_log", ovnControllerLogPath}...)
+	if err != nil {
+		return false, fmt.Errorf("error accessing logs in node %s: %v", targetNodeName, err)
+	}
+	return verifyDenyAuditLogs(targetNodeLog, namespace, policyName, aclSeverity), nil
+}
+
+func makeDenyAllPolicy(f *framework.Framework, ns string, policyName string) (*knet.NetworkPolicy, error) {
+	policy := &knet.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: policyName,
+		},
+		Spec: knet.NetworkPolicySpec{
+			PodSelector: metav1.LabelSelector{},
+			PolicyTypes: []knet.PolicyType{knet.PolicyTypeEgress, knet.PolicyTypeIngress},
+			Ingress:     []knet.NetworkPolicyIngressRule{},
+			Egress:      []knet.NetworkPolicyEgressRule{},
+		},
+	}
+	return f.ClientSet.NetworkingV1().NetworkPolicies(ns).Create(context.TODO(), policy, metav1.CreateOptions{})
+}
+
+func pokePod(fr *framework.Framework, srcPodName string, dstPodIP string) error {
```

Comment:
I would move this to a more standard place for all e2e tests to use like  https://github.com/ovn-org/ovn-kubernetes/blob/master/test/e2e/util.go